FutureCollars/src/main/java/lesson11task1/DivisibilityValidator.java

```diff
@@ -0,0 +1,17 @@
+package lesson11task1;
+
+import java.util.IntSummaryStatistics;
```

Comment:
Tu chyba przypadkowo dodał się plik do commita, zwracaj proszę uwagę na to co commitujesz na konkretne branche, łatwiej będzie zachować porządek w projekcie :)